sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/models/KeyWrapAlgorithm.java

```diff
@@ -43,4 +43,18 @@ public String toString() {
      */
     public static final List<KeyWrapAlgorithm> ALL_ALGORITHMS = Collections
             .unmodifiableList(Arrays.asList(RSA_OAEP, RSA1_5, RSA_OAEP_256, A192KW, A128KW, A256KW));
+
+    /**
+     * Return the KeyWrapAlgorithm which maps to {@code value}
+     * @param value The value whose equivalent KeyWrapAlgorithm is needed.
+     * @return the KeyOperation
+     */
+    public static KeyWrapAlgorithm fromString(String value) {
```

Comment:
Is this meant to be similar to the `ExpandableStringEnum` class functionality in Azure Core or is this meant to handle de-serialization of `KeyWrapAlgorithm`s? If it is the latter would we want this class to extend `ExpandableStringEnum` to better handle changes in the response values from services?

Reply:
Yeah, I am making this change in a separate PR.